conf/solution/config.yaml.sample

```diff
@@ -81,6 +81,7 @@ cortx:
                                                                # Ports: 21002 - (21002 + Number of CVGs)
       - tcp://data2-node1:21002                                # Same thing for the second pod
       - tcp://data1-node2:21002                                # of for Pods on the other nodes
+      - tcp://data2-node2:21002
```

Comment:
Guess we need to add 2 more entries:
`- tcp://data1-node3:21002`
`- tcp://data2-node3:21002`

Reply:
done, Updated this in all files now.